src/platform/posix.c

```diff
@@ -479,23 +548,51 @@ void platform_enumerate_directories_end(int handle)
 }
 
 int platform_get_drives(){
-	/*
-	return GetLogicalDrives();
-	*/
-	STUB();
+	// POSIX systems do not know drives. Return 0.
 	return 0;
 }
 
 bool platform_file_copy(const utf8 *srcPath, const utf8 *dstPath, bool overwrite)
 {
-	STUB();
+	log_verbose("Copying %s to %s", srcPath, dstPath);
+
+	// If overwrite is not on and the file already exists, return 0
+	if(!overwrite && platform_file_exists(dstPath)) {
```

Comment:
This could introduce race conditions, and checking for file existence is an anti-pattern. Instead, how about opening the file for writing with the `x` flag. This subspecifier forces the function to fail if the file exists, instead of overwriting it.

You can then check for `EEXIST` in the `errno` to see if the error was because the file already existed.